content/docs/editing/mdx.md

```diff
@@ -1,167 +1,51 @@
 ---
-title: Rich text with MDX
+title: Markdown & MDX
```

Comment:
Might make more sense to have the name of this file be "markdown" rather than "mdx"?